src/main/java/com/google/devtools/build/lib/actions/PathStripper.java

```diff
@@ -72,182 +74,139 @@
  */
 public final class PathStripper {
   /**
-   * Support for stripping config paths from an action's inputs and outputs.
+   * Creates a new {@link PathMapper} for action implementation logic to use.
```

Comment:
I'm a bit confused by the wording at the end: "action implementation logic to use". Especially where right under `class PathStripper` it mentions `creates a PathMapper`. 

Are you intending further refactoring here? If so, no worries. If not I'd try to be more precise.

Reply:
Yes, I would propose a further change as a follow-up that would clean up this comment: Make `PathMapper` a regular top-level class defined in a separate file and entangle the particular implementations created by `PathStripper` from it.